app/src/main/java/com/sunlightlabs/congress/services/LegislatorService.java

```diff
@@ -104,6 +120,9 @@ else if (!json.isNull("id"))
         if (!json.isNull("in_office"))
             legislator.in_office = json.getBoolean("in_office");
 
+        if (!json.isNull("at_large"))
+            legislator.at_large = json.getBoolean("at_large");
```

Comment:
This is present now when fetching individual members, and used to display when someone is At-Large. It may not be in member lists yet. Tracking at https://github.com/propublica/congress-api-docs/issues/41